README.md

```diff
@@ -11,7 +11,20 @@
 
 <a href="https://alpha.twizzle.net/"><img src="./src/sites/alpha.twizzle.net/twizzle-social-media-image.png" width="256">
 
-Twizzle</a> is based on `cubing.js`, and is currently being developed at [`src/sites/alpha.twizzle.net`](./src/sites/alpha.twizzle.net/).
+Twizzle</a> is the spiritual successor to [alg.cubing.net](https://alg.cubing.net/), based on `cubing.js`. It is currently being developed at [src/sites/alpha.twizzle.net](./src/sites/alpha.twizzle.net/).
+
+See the [Twizzle Diaries](https://www.youtube.com/watch?v=9_kqXn0Mq-o&list=PLFh3NgpDbzN4VkcfjEZSQ_TYQv_OEjbjF) video series for more information on Twizzle's vision and use cases.
+
+## Usage
+
+If you would like to use cubing.js as a library in your projects, make sure you have [node](node.js) installed.
+
+With Node installed, run the following command in your terminal:
+
+```shell
+npm install cubing
+```
+
```

Comment:
I've been holding off on adding this because:

1) the library is not quite stable enough in some places, and
2) if someone doesn't already know how to use `node` and `npm`, we should steer them to using `cdn.cubing.net` directly in a web page.

Hopefully 1) will become less of a problem with documentation.

Would you mind moving this under the current `Documentation` section and renaming them something like this?

- `Documentation` → `Getting started`
- `Usage` → `Using `

If we're showing how to install `cubing`, I think we should also provide some very basic code snippets like "Using `npm`" at https://js.cubing.net/cubing/#javascript

Eventually I'd like to make some videos on how to do this more easily, but for now I'd like to emphasize "if you're already comfortable writing web apps using `npm`, here's how; if you don't already know, you should definitely start with `cdn.cubing.net`".

Reply:
Ok, let me know how the new sections look. They probably need another revision, but hopefully they're closer to the mark. "Documentation" → "Getting started and "Usage" → "Using with Node.js", on the assumption that only people who know they're looking to use it in a Node project will be attracted there. 

I just copy-pasted the import statements from https://js.cubing.net/cubing/#javascript as code snippets - let me know if you want me to (try) add something more substantial.